### PR TITLE
Important addition

### PR DESCRIPTION
**Pull request by EmilHolmer**
Very important addition.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,6 @@ author: "Marsja"
 date: "11/10/2021"
 ---
 
-Stuff!
\ No newline at end of file
+Stuff!
+
+A lot of stuff!
```
Comment (EmilHolmer) [inline README.md:9]: See this very important addition.

**Review by EmilHolmer — COMMENTED** [review]
Super-important.

File: README.md (hunk 1 above)
Comment (marsja) [reply README.md:9]: I can see how important this is.

**Review by marsja — APPROVED** [review]
I think we can accept this. It seems so important

**Pull request merged**